tox.ini

```diff
@@ -105,3 +104,5 @@ commands = coverage erase
 skip_install = true
 deps = coverage
 
+[flake8]
+ignore = E226,E302,E41,E122, E241
```

Comment:
Ah, the old "fix by stop checking" fix :)

It's really up to you if you want to maintain strict PEP8 syntax or not. This was part of the package building script I ran, I don't feel strongly about it either way. If you're going to comment out a certain class of errors, it might be worth just turning this off in general; PEP8 compatibility only checks code format, not correctness, so nothing is gained that unit tests don't already pick up.